frameworks/PHP/workerman/workerman-pgsql.dockerfile

```diff
@@ -11,17 +13,13 @@ RUN apt-get install -yqq php8.3-cli php8.3-pgsql php8.3-xml > /dev/null
 COPY --from=composer/composer:latest-bin --link /composer /usr/local/bin/composer
 
 RUN apt-get install -y php-pear php8.3-dev libevent-dev git > /dev/null && \
-    pecl install event-3.1.3 > /dev/null && echo "extension=event.so" > /etc/php/8.3/cli/conf.d/event.ini
-
-COPY --link php.ini /etc/php/8.3/cli/php.ini
+    pecl install event-3.1.3 > /dev/null && echo "extension=event.so" > /etc/php/8.3/cli/conf.d/30-event.ini
```

Comment:
The last event extension is 3.1.4

Reply:
Perhaps we can directly use the command `pecl install event`, which will install the latest version of the event.

Reply:
Before I used event without version, but sometimes had problems.

Reply:
But you can use it if you want.

Reply:
Another advantage, it's that so we can check the performance of the event extension when change.
And so we know if a performance change in Workerman or in event.

Reply:
Yes, you are right.